docs/modules/ROOT/pages/guide-metatx.adoc

```diff
@@ -0,0 +1,349 @@
+
+This https://github.com/OpenZeppelin/workshops/tree/master/25-defender-metatx-api[demo app] implements metatransactions using https://docs.openzeppelin.com/contracts/4.x/api/metatx#MinimalForwarder[MinimalForwarder] and https://docs.openzeppelin.com/contracts/4.x/api/metatx#ERC2771Context[ERC2771Context] to separate `msg.sender` from the relayer's address. All the user needs to do is sign a message using the account they would like to issue the transaction from. The signature is formed from the user's private key and the data of the desired transaction. This signing happens off-chain and costs no gas. The signature is passed to the Relayer so it can execute the transaction for the user (and pay the gas).
+
+[[demo-app-overview]]
+== Demo App Overview
+
+You can view the live https://defender-metatx-workshop-demo.openzeppelin.com/[demo dapp] here. It accepts registrations directly if the user has the available funds to pay for the transaction, otherwise the data is sent as a meta-transaction.
+
+In the example code, the functionality of the https://github.com/OpenZeppelin/workshops/blob/master/25-defender-metatx-api/contracts/SimpleRegistry.sol[`SimpleRegistry` contract] is to take a string and store it. The contract's https://github.com/OpenZeppelin/workshops/blob/master/25-defender-metatx-api/contracts/Registry.sol[metatransaction implementation] achieves the same result by decoupling the signer from the sender of the transaction.
+
+When comparing the code, note the metatransaction's use of `_msgSender()` as opposed to the SimpleRegistry's use of `msg.sender`. By extending from `ERC2771Context` and `MinimalForwarder`, the contract becomes metatransaction capable. (Note that all OpenZeppelin contracts are compatable with the use of `_msgSender()`.)
+
+The second fundamental change between the two contracts is the need for the metatransaction contract to specify the address of the trusted forwarder, which in this case is the address of the `MinimalForwarder` contract.
+
+[[configure-project]]
+== Configure the project
+
+First, fork the repository and then `git clone` your fork to your computer and install dependencies
+
+```
+$ git clone https://github.com/[GitHub username]/workshops.git
+$ cd workshops/25-defender-metatx-api/
+$ yarn
+```
+
+Create a `.env` file in the project root and supply your Team API key and secret from Defender. A private key will be used for local testing but the Relayer is used for actual contract deployment.
+
+```
+PRIVATE_KEY="Goerli private key"
+TEAM_API_KEY="Defender Team API key, used for uploading autotask code"
+TEAM_API_SECRET="Defender Team API secret"
+```
+
+[[create-relayer]]
+== Create Relayer
+
+First, install the necessary package:
+
+`npm install defender-relay-client`
+
+Create a script such as the following, supplying your Defender API key in your .env file and adjusting variable names as appropriate.
+
```

Comment:
I'd include the script in the repo, so they don't have to copy-paste it from here. We can include in the guide only the relevant parts (ie the lines that actually create the relayer)